charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -39,24 +39,51 @@ spec:
       serviceAccountName: machine-controller-manager
       terminationGracePeriodSeconds: 5
       containers:
-      - name: azure-machine-controller-manager
-        image: {{ index .Values.images "machine-controller-manager" }}
+      - name: machine-controller-manager-provider-azure
```

Comment:
You need to extend the logging configuration here https://github.com/gardener/gardener-extension-provider-azure/blob/master/charts/gardener-extension-provider-azure/templates/configmap-logging.yaml to collect properly the logs from the newly added container

Reply:
Hi @vpnachev ,

With the previous PRs for migration to OOT for other providers, I didn't have to make changes to collect logs from the new containers. My guess is that [this configuration](https://github.com/gardener/gardener-extension-provider-azure/blob/master/charts/gardener-extension-provider-azure/templates/configmap-logging.yaml#L19) already collects the logs?

Refer AWS log collection templates as well [here](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/configmap-logging.yaml).

Reply:
This configuration https://github.com/gardener/gardener-extension-provider-azure/blob/4159e890852a5ccc9020502aba7953be6567dd4d/charts/gardener-extension-provider-azure/templates/configmap-logging.yaml#L19 matches only the other container 

/invite @Kristian-ZH @vlvasilev